reference/legacy-upgrade.md

```diff
@@ -1,6 +1,6 @@
 ## Legacy Upgrade Support
 
-A legacy installation of Mayastor (1.0.5 and below) cannot be seamlessly upgraded and needs manual intervention.
+A legacy installation of Mayastor (1.0.x and below) cannot be seamlessly upgraded and needs manual intervention.
 Follow the below steps if you wish to upgrade from Mayastor 1.0.x to Mayastor 2.1.0 and above.
```

Comment:
Should this be 2.x.y here?
CC @avishnu

Reply:
The support is present only from 2.1 onwards.